### PR TITLE
New launch form updates

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Document the new launch form and update launch advanced options page. DO NOT MERGE BEFORE NEW LAUNCH FORM IS RELEASED.

### CODE REVIEW

File: platform_versioned_docs/version-23.4.0/launch/launchpad.mdx [hunk 1]
```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
...
 
-The **Launchpad** enables workspace users to launch pre-configured pipelines, add new pipelines, or perform a quick launch of unsaved pipelines. Use the **Sort by:** drop-down to sort pipelines, either by name or most-recently updated.
+The **Launchpad** enables workspace users to launch pre-configured pipelines, add new pipelines, or perform a quick launch of unsaved pipelines. Use the **Sort by:** dropdown to sort pipelines, either by name or most-recently updated.
 
 :::note
 A pipeline is a repository containing a Nextflow workflow, a compute environment, and pipeline parameters.
 :::
 
-The list layout is the default **Launchpad** view. Use the toggle to the right of the **Search** field to switch between the list and tile views. Both views display the compute environment of each pipeline for easy reference.
+The list layout is the default **Launchpad** view. Use the toggle next to the **Search** field to switch between the list and tile views. Both views display the compute environment of each pipeline for easy reference.
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.4.0/launch/launchpad.mdx:18]: @llewellyn-sl, this is one of the few areas where access to inline SVGs might be helpful.

File: platform_versioned_docs/version-23.4.0/launch/launchpad.mdx [hunk 2]
```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
...
 
 ## Launch form
 
-The **Launch form** is used to launch pipelines and to add pipelines to the Launchpad. Select **Launch** next to a saved pipeline in the Launchpad list, or select **launch a run without configuration** below the Launchpad heading to perform a quick launch of an unsaved pipeline.
+The launch form is used to launch pipelines and to add pipelines to the **Launchpad**. Select **Launch** next to a saved pipeline in the list, or select **launch a run without configuration** to perform a quick launch of an unsaved pipeline.
 
-**Launch a saved pipeline**
+The launch form consists of ([General config](#general-config), [Run parameters](#run-parameters), and [Advanced options](#advanced-options)) sections to specify your run parameters before execution, and an execution summary. Use section headings or select the **Previous** and **Next** buttons at the bottom of the page to navigate between sections.
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.4.0/launch/launchpad.mdx:24]:
```suggestion
The launch form consists of [General config](#general-config), [Run parameters](#run-parameters), and [Advanced options](#advanced-options) sections to specify your run parameters before execution, and an execution summary. Use section headings or select the **Previous** and **Next** buttons at the bottom of the page to navigate between sections. 
```

File: platform_versioned_docs/version-23.4.0/launch/launchpad.mdx [hunk 3]
```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
...
+The launch form consists of ([General config](#general-config), [Run parameters](#run-parameters), and [Advanced options](#advanced-options)) sections to specify your run parameters before execution, and an execution summary. Use section headings or select the **Previous** and **Next** buttons at the bottom of the page to navigate between sections. 
 
-Select **Launch** next to the pipeline of your choice to open the pipeline launch form.
+For saved pipelines, **General config** and **Run parameters** fields are prefilled and can be edited before launch. 
 
-Seqera uses a `nextflow_schema.json` file in the root of the pipeline repository to dynamically create a form with the necessary pipeline parameters.
+### General config 
 
-All pipelines contain at least these parameters:
+- **Pipeline to launch**: A Git repository name or URL. For saved pipelines, this is prefilled and cannot be edited. Private repositories require [access credentials](../credentials/overview.mdx).
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.4.0/launch/launchpad.mdx:30]: I always wonder whether it is Git or git. Based on this it is Git: https://github.com/swcarpentry/git-novice/issues/505

File: platform_versioned_docs/version-23.4.0/launch/launchpad.mdx [hunk 4]
```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
...
 
-    :::note
-    In YAML, quotes should be used for paths but not for numbers or Boolean values.
-    :::
+Use resource labels to tag the computing resources created during the workflow execution. While resource labels for the run are inherited from the compute environment and pipeline, admins can override them from the launch form. Applied resource label names must be unique. 
 
+#### Pipeline secrets 
 
-1. Select or create [resource labels](../resource-labels/overview.mdx) to apply to the resources consumed by the run, if needed.
+Secrets are used to store keys and tokens used by workflow tasks to interact with external systems. Enter the names of any stored user or workspace secrets required for the workflow execution.
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.4.0/launch/launchpad.mdx:80]:
```suggestion
Secrets are used to store keys and tokens used by workflow tasks to interact with external systems. Enter the names of any stored user or workspace secrets required for the workflow execution.
```

File: platform_versioned_docs/version-23.4.0/launch/launchpad.mdx [hunk 5]
```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
...
+#### Pipeline secrets 
 
-1. Select or create [resource labels](../resource-labels/overview.mdx) to apply to the resources consumed by the run, if needed.
+Secrets are used to store keys and tokens used by workflow tasks to interact with external systems. Enter the names of any stored user or workspace secrets required for the workflow execution. 
 
-1. Expand the [Pipeline secrets](../secrets/overview.mdx) dropdown to select user or workspace secrets to be used during the run, if needed.
+#### Advanced options
 
-1. Apply any [Advanced options](./advanced.mdx), if needed.
+See [Advanced options](../launch/advanced.mdx).
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.4.0/launch/launchpad.mdx:84]:
```suggestion
See [Advanced options](../launch/advanced.mdx).
```

**Review by jason-seqera — APPROVED** [review]
@llewellyn-sl, left a few minor comments, thanks!

**Comment by llewellyn-sl** [comment]
Replaced by https://github.com/seqeralabs/docs/pull/80. Closing.

**Pull request closed without merging**